BUILD.bazel

```diff
@@ -0,0 +1,337 @@
+load(
+    "//ign_bazel:cmake_configure_file.bzl",
+    "cmake_configure_file",
+)
+load(
+    "//ign_bazel:generate_include_header.bzl",
+    "generate_include_header",
+)
+load(
+    "//ign_bazel:ign_export_header.bzl",
+    "ign_export_header",
+)
+load(
+    "//ign_bazel:generate_yaml.bzl",
+    "generate_yaml",
+)
+load(
+    "//ign_bazel:qt.bzl",
+    "qt_resource",
+    "qt_cc_library",
+    "qt_cc_binary",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+public_headers_no_gen = [
+    "//ign_gui/include/ignition/gui:Application.hh",
```

Comment:
This seems really strange, is the relative path not sufficient here?

Reply:
Long story short, a build file must exist at the level of a qt resource `.qrc` file, since one exists in `include/ignition/gui`, the higher level build file sees the headers as being owned by the lower level build file, hence the out of the ordinary referencing.